### DISCUSSION TITLE
error 404 when refreshing on azure

### DISCUSSION THREAD

**Discussion by thomas159** [vercel/next.js#20810] [Help]
I am hosting nextjs on an azure server and using the static export  to build. ` "build": "next build && next export",`

However when refreshing the site it goes to 404 not found,

Is there something I need to add the the next.config.js or other config that needs to be done to get this working correctly?

**Comment by bendandoSolo** [comment]
Having the same error!!

**Comment by joraanbe82** [comment]
If yours routes are statics, you have to define into next.config.js the exportPathMap 
[here](https://nextjs.org/docs/api-reference/next.config.js/exportPathMap) is the documentation.
